### PR TITLE
fix(#23): add dark mode shortcut + fix buttons

### LINKED ISSUE #23: Fix dark buttons
Dark link buttons on [`archive.html`](https://github.com/genicsblog/theme-files/blob/main/_layouts/archive.html#L54) look weird:

image

Needs to be similar style as the [CTA](https://github.com/genicsblog/theme-files/blob/main/_includes/cta.html#L12) (size should be same, needs fix just for the shadow and colour):

image

### PR DESCRIPTION
**Pull request by carrotfarmer**
- add `ctrl/cmd + shift + l` shortcut for light/dark mode
- fix "Next" and "Previous" buttons in `archive.html`

fixes #23

### CODE REVIEW

File: _layouts/archive.html [hunk 1]
```diff
@@ -1,78 +1,84 @@
...
+        >
+          {% for post in paginator.posts %} {% include postbox.html %} {% endfor
+          %}
+        </div>
+        {% if paginator.total_pages > 1 %}
+        <div class="flex justify-center items-center w-full">
+          {% if paginator.previous_page %}
+          <a
+            href="{{ paginator.previous_page_path }}"
+            class="mr-2 px-4 ml-2 py-0.5 font-bold rounded-md text-black bg-white hover:bg-slate-300 hover:scale-110 transition underline-none"
```
Comment (gouravkhunger) [inline _layouts/archive.html:57]: You changed the light one too :)
Light one is correct, just need to fix `dark:` values.

image

File: _layouts/archive.html [hunk 2]
```diff
@@ -1,78 +1,84 @@
...
+          >
+            Previous
+          </a>
+          {% endif %}
+          <span>Page {{ paginator.page }} of {{ paginator.total_pages }}</span>
+          {% if paginator.next_page %}
+          <a
+            href="{{ paginator.next_page_path }}"
+            class="ml-2 px-4 py-0.5 font-bold rounded-md text-black bg-white hover:bg-slate-300 hover:scale-110 transition underline-none"
```
Comment (gouravkhunger) [inline _layouts/archive.html:66]: same here.

File: assets/js/theme.js [hunk 3]
```diff
@@ -104,6 +104,12 @@ const toggleTheme = () => {
   }
 };
 
+document.addEventListener("keydown", function (e) {
+  if ((e.ctrlKey || e.metaKey) && e.shiftKey && e.keyCode === 76) {
```
Comment (gouravkhunger) [inline assets/js/theme.js:108]: is `ctrl/command + shift + l` generally the default keybinding for dark mode? I am not sure

**Review by gouravkhunger — CHANGES_REQUESTED** [review]
Left some comments :)

File: assets/js/theme.js (hunk 3 above)
Comment (carrotfarmer) [reply assets/js/theme.js:108]: lol it's `cmd + shift + l` in notion, so I assumed it would be like this in other apps. I'm not sure.

**Review by gouravkhunger — APPROVED** [review]
Thanks!

**Pull request merged**